docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md

```diff
@@ -129,10 +131,10 @@ cd docker-images/OracleSOASuite/dockerfiles
 ./buildDockerImage.sh -v 12.2.1.3 -s
 ```
 
-The image produced will be named `middleware/soasuite/oracle/soasuite:12.2.1.3`.
+The image produced will be named `localhost/oracle/soasuite:12.2.1.3`.
```

Comment:
that is not a good name for an image..

Reply:
Agreed.  Why localhost?

Reply:
The image name is constructed as "IMAGE_NAME="${DC_REGISTRY_SOA}/oracle/soasuite:$VERSION"" in file "https://github.com/oracle/docker-images/blob/master/OracleSOASuite/dockerfiles/buildDockerImage.sh".

And export DC_REGISTRY_SOA="localhost" is set in "https://github.com/oracle/docker-images/blob/master/OracleSOASuite/setenv.sh".

Hence mentioned the users to re-tag the built image. Please review.

Reply:
That's going to be very confusing to customers...  Maybe fix it in docker-images?